### PR TITLE
Fixed undoing twice when removing selection and 'complex' typo in method name

### PR DESCRIPTION
**Pull request by Paulb23**
Fixed having to undo twice when a selection is removed as mentioned [here.](https://github.com/godotengine/godot/pull/4193)

### CODE REVIEW

File: scene/gui/text_edit.cpp [hunk 1]
```diff
@@ -1747,6 +1747,7 @@ void TextEdit::_input_event(const InputEvent& p_input_event) {
 				}
 				if (clear) {
 
+					_begin_compex_operation();
```
Comment (akien-mga) [inline scene/gui/text_edit.cpp]: Maybe we should fix the typo in this method's name :P

Comment (Paulb23) [inline scene/gui/text_edit.cpp]: I never noticed :P shall I add it to this PR?

Comment (akien-mga) [inline scene/gui/text_edit.cpp]: Sure, why not. Keep it in a separate commit though.
